chimney/src/main/scala/io/scalaland/chimney/TransformerFSupport.scala

```diff
@@ -0,0 +1,93 @@
+package io.scalaland.chimney
+
+import scala.collection.compat._
+
+trait TransformerFSupport[F[_]] {
+
+  def pure[A](value: A): F[A]
+
+  def combine[A, B](fa: F[A], fb: => F[B]): F[(A, B)]
+
+  def map[A, B](fa: F[A], f: A => B): F[B]
+
+  def traverse[M, A, B](it: Iterator[A], f: A => F[B])(
```

Comment:
In particular we need to explain that here we use `Iterator` only **internally** at it doesn't impose limitations of what structures can be handled by derivation.

Reply:
explained